### PR TITLE
fix(ui): load recents prj from localStorage

### PR DESCRIPTION
**Pull request by yesnault**
And use subscription instead of hard cleanCache on navbar

Signed-off-by: Yvonnick Esnault <yvonnick@esnau.lt>

### CODE REVIEW

File: ui/src/app/store/project.state.ts [hunk 1]
```diff
@@ -441,7 +441,7 @@ export class ProjectState {
                 project_key: action.payload.projectKey
             }
         ).pipe(tap(() => {
-            this._navbarService.getData(); // TODO: to delete
+            this._navbarService.refreshData(); // TODO: to delete
```
Comment (sguiheux) [inline ui/src/app/store/project.state.ts]: remove todo?

File: ui/src/app/store/workflow.state.ts [hunk 2]
```diff
@@ -677,7 +677,7 @@ export class WorkflowState {
                 workflow_name: action.payload.workflowName,
             }
         ).pipe(tap(() => {
-            this._navbarService.getData(); // TODO: to delete
+            this._navbarService.refreshData(); // TODO: to delete
```
Comment (sguiheux) [inline ui/src/app/store/workflow.state.ts]: remove todo?

File: ui/src/app/store/project.state.ts (hunk 1 above)
Comment (yesnault) [inline ui/src/app/store/project.state.ts]:
```suggestion
            this._navbarService.refreshData();
```

File: ui/src/app/store/workflow.state.ts (hunk 2 above)
Comment (yesnault) [inline ui/src/app/store/workflow.state.ts]:
```suggestion
            this._navbarService.refreshData();
```

**Pull request merged**